src/cata_tiles.cpp

```diff
@@ -3827,7 +3838,8 @@ void cata_tiles::draw_zlevel_overlay( const tripoint &p, const lit_level ll, int
         fog_color = curses_color_to_SDL( c_dark_gray );
     }
     // Setting for fog transparency
-    fog_color.a = 100;
+    // On isometric tilesets, fog intensity scales with zlevel_height in tile_config.json
+    fog_color.a = is_isometric() ? 50 + zlevel_height / 2 : 100;
```

Comment:
This should be capped at 255.

(Or changed to `255.0 - 255.0 * pow(155.0 / 255.0, (z_level_height / some_reference_value))`, if we want it to be mathematically accurate. It should be cached in that case to improve speed though.)

Reply:
Good catch.  Thanks!